playwright/tests/a11y.spec.js

```diff
@@ -1,66 +1,148 @@
-  for (const pageUrl of pages) {
-    await page.goto(pageUrl)
-    const accessibilityScanResults = await makeAxeBuilder().analyze()
-
-    if (accessibilityScanResults.violations.length > 0) {
-      accessibilityScanResults.violations.forEach((violation) => {
-        console.log(`URL: ${pageUrl}`)
-        console.log(`Violation: ${violation.id}`)
-        console.log(`Description: ${violation.description}`)
-        console.log(`Impact: ${violation.impact}`)
-        console.log(
-          `Nodes: ${violation.nodes.map((node) => node.html).join(', ')}`
-        )
-      })
-
-      a11yFailures.push({
-        url: pageUrl,
-        violations: accessibilityScanResults.violations,
-      })
-    }
-  }
+  splitPagesIntoBatches,
+} from '../utils/getSitemapLocations'
+import AxeBuilder from '@axe-core/playwright'
+import fs from 'fs'
 
-  if (a11yFailures.length > 0) {
-    throw new Error(
-      `Accessibility tests (${testName}) failed on ${a11yFailures.length} pages. Check logs for details.`
-    )
-  }
-}
+const baseUrl = process.env.BASE_URL || 'http://127.0.0.1:8001'
+const totalSegments = process.env.TOTAL_SEGMENTS
+  ? Number(process.env.TOTAL_SEGMENTS)
+  : 32
+const segmentNumber = process.env.SEGMENT_INDEX
+  ? Number(process.env.SEGMENT_INDEX)
+  : 0
+const excludedPages = [
+  'http://www.va.gov/manage-va-debt/summary/',
+  'http://www.va.gov/housing-assistance/home-loans/check-coe-status/your-coe/',
+  'http://www.va.gov/education/download-letters/letters/',
+  'http://www.va.gov/health-care/appointment-pre-check-in/',
+  'http://www.va.gov/resources/search/',
+  'http://www.va.gov/va-payment-history/payments/',
+  'http://www.va.gov/records/get-veteran-id-cards/apply/',
+  'http://www.va.gov/health-care/order-hearing-aid-or-CPAP-supplies-form/',
+  'http://www.va.gov/education/gi-bill/post-9-11/ch-33-benefit/status/',
+  'http://www.va.gov/housing-assistance/home-loans/request-coe-form-26-1880/',
+  'http://www.va.gov/records/download-va-letters/letters/',
+  'http://www.va.gov/decision-reviews/board-appeal/request-board-appeal-form-10182/',
+  'http://www.va.gov/burials-memorials/memorial-items/presidential-memorial-certificates/request-certificate-form-40-0247/',
+  'http://www.va.gov/my-health/order-hearing-aid-or-cpap-supplies-form/',
+  'http://www.va.gov/education/other-va-education-benefits/veteran-rapid-retraining-assistance/apply-for-vrrap-form-22-1990s/',
+  'http://www.va.gov/track-claims/',
+  'http://www.va.gov/burials-and-memorials-v2/application/530/',
+  'http://www.va.gov/profile/',
+  'http://www.va.gov/my-va/',
+  'http://www.va.gov/disability/view-disability-rating/rating/',
+  'http://www.va.gov/view-change-dependents/view/',
+  'http://www.va.gov/supporting-forms-for-claims/substitute-claimant-form-21P-0847/',
+  'http://www.va.gov/supporting-forms-for-claims/alternate-signer-form-21-0972/',
+  'http://www.va.gov/careers-employment/education-and-career-counseling/apply-career-guidance-form-25-8832/',
+]
```

Comment:
let's also add:
```
'http://www.va.gov/js/smartbanner/README/',
'http://www.va.gov/js/smartbanner/LICENSE/'
```

why these two are in the sitemap in the first place is a different story, but let's not scan them. they always report non-actionable errors